ecospheres_migrator/migrator.py

```diff
@@ -69,15 +71,24 @@ def transform(self, transformation: Path, selection: list[Record]) -> TransformB
 
         batch = TransformBatch()
         for r in selection:
-            log.debug(f"Processing {r.uuid} (template={r.template} state={r.state})")
+            log.debug(f"Processing {r.uuid} (md_type={r.md_type.name} state={r.state})")
```

Comment:
```suggestion
            log.debug(f"Processing record {r.uuid}: md_type={r.md_type.name}, state={r.state}")
```
To be consistent with `log.debug(f"Updating record {uuid}: md_type={md_type.value}, state={state}")` in `put_record()`